### ISSUE TITLE
Validation Error - Message Header

### ISSUE THREAD

**Issue by rsethu** [ewoutkramer/fhir-net-api#328]
The message header's TimestampElement property when set with a valid value like 
`TimestampElement = new Instant(DateTimeOffset.UtcNow)`
yeilds an error stating 

> " is not a correctly formatted Instant" as Error Message

> "Value" is the Member Name

**Comment by brianpos** [comment]
Can you confirm your regional setting information for times?
And if possible contribute a unit test that shows the failure, makes it much easier to know when we have the fix right.

**Comment by rsethu** [comment]
(UTC-08.00) Pacific Time (US & Canada)

```
var results = new List<ValidationResult>();
var isValid = DotNetAttributeValidation.TryValidate(new MessageHeader()
{
	Event = new Coding("system-test", "some-query"),
	TimestampElement = new Instant(DateTimeOffset.UtcNow),
	Source = new MessageHeader.MessageSourceComponent
	{
		Name = "TestProvider",
		Software = "TestSoftware",
		Endpoint = Guid.NewGuid().ToString(),
		EndpointElement = new FhirUri("testurl")
	}
}, results, true);

Assert.IsTrue(isValid);
Assert.That(results.Count, Is.EqualTo(0))
```

**Comment by brianpos** [comment]
Thanks will have a look at it.

**Comment by rsethu** [comment]
Still the stable version 0.93.4.0 validation yields an error

**Comment by ewoutkramer** [comment]
Yes - this is probably still open.

**Comment by ewoutkramer** [comment]
Found the cause: we actually had a [InstantPattern] attribute - back from when instant was a string (if ever) - anyway, nowadays all Instants are represented as DateTimeOffset in the Poco, so validating it against a "string" pattern using a regex (which is what InstantPattern was) is silly and it would always return an error.

**Comment by ewoutkramer** [comment]
Fixed in 51c0291
